docs/tooling_and_ecosystem/generators/matestack_page_generator.md

```diff
@@ -0,0 +1,52 @@
+# Matestack Page Generator
+
+## Usage
+
+Generates matestack pages to `app/matestack/pages`.
+
+## Parameters
+
+**NAME** - Mandatory. Creates an empty matestack component in `app/matestack/components/NAME.rb`.
+
+**--app_name** - Mandatory. Adds a `/app_name` folder to `app/matestack/pages` to indicate to which matestack app the pages belongs. Pages _do not necessarily need to belong to an app_, but for now the page scaffolder can only provide this functionality.
+
+**--namespace** - Optional. Adds a `/namespace` folder within the `/app_name` and a namespace to the `page.rb`.
```

Comment:
how do I created deeper namespaces e.g. `app/matestack/pages/my
_app/namespace_1/namespace_2/my_page.rb`? couldn't this be accomplished by just using NAME with "/" in it?

Reply:
Yeah it can be accomplished the way you said. I'll add it to the docs